using-mobile-extensions/mobile-core/configuration/configuration-api-reference.md

```diff
@@ -101,6 +101,10 @@ You can also update the configuration programmatically by passing configuration
 Keys that are not found on the current configuration are added when this method is followed. Null values are allowed and replace existing configuration values.
 {% endhint %}
 
+{% hint style="warning" %}
+Should never use this API to update `build.environment` or any key with environment prefix, it could lead to unexpected behaviors. For more information, please read [Configure with Launch App ID](./configuration#environment-aware-configuration-properties).
```

Comment:
Reword the first sentence so that it reads as follows:

"Do not use this API to update `build.environment` or any key with environment prefix, because using this API can lead to unexpected behaviors."